src/store/traction/pacbio/runCreate/getters.js

```diff
@@ -9,4 +17,34 @@ export default {
   defaultSmrtLinkVersion: (state, getters) => {
     return Object.values(getters.smrtLinkVersionList).find((version) => version.default)
   },
+
+  // copied over from pools/getters.js
```

Comment:
If we copied this over verbatim then it sounds like as as part of the refactor we should reuse this function.

Reply:
This will also hopefully be resolved during the refactoring https://github.com/sanger/traction-ui/issues/941